components/admin_console/request_button/request_button.tsx

```diff
@@ -165,7 +171,8 @@ export default class RequestButton extends React.PureComponent {
             success: false,
         });
 
-        const doRequest = () => { //eslint-disable-line func-style
+        const doRequest = () => {
+            //eslint-disable-line func-style
```

Comment:
Since moving this onto a new line didn't change anything, I think we might be able to just remove it